script/setup

```diff
@@ -31,7 +31,7 @@ pushd "${base_dir}/parser" 2>/dev/null
 pip3 install -e '.'
 popd 2>/dev/null
 
-pip3 install -r "${base_dir}/requirements.txt"
+pip3 install -r "${base_dir}/requirements_dev.txt"
```

Comment:
Why split the requirements file, when you are just referencing the combined again here?

Reply:
This allow simpler management of linters and doesn't force lib users to install linters, I don't mind combining it back if you think it is better

Reply:
Lib users? This is not a lib?
We should eventually (with pre-commit) force linting anyway.

Reply:
Ok, I will combine back, pre-commit was my next planned PR anyhow